test/__snapshots__/test.js.snap

```diff
@@ -1,5 +1,75 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`with processors AsyncCallbackInterface.webidl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+const interfaceName = \\"AsyncCallbackInterface\\";
+
+exports.convert = function convert(value, { context = \\"The provided value\\" } = {}) {
+  if (!utils.isObject(value)) {
+    throw new TypeError(\`\${context} is not an object.\`);
+  }
+
+  const { wrapper } = {
+    wrapper(...args) {
+      let completion;
+      let thisArg = this;
+      let O = value;
+      let X = O;
+
+      try {
```

Comment:
Interesting strategy. You're essentially inlining https://heycam.github.io/webidl/#call-a-user-objects-operation into the conversion process, so after we go through the Web IDL bindings, the callback function is essentially represented by a specific "call the user's operation" function for this particular callback. This doesn't follow the spec, but is probably easier for jsdom to deal with. Still, I wonder if there's a way we could make this more obvious. Maybe just documentation...

Reply:
One strategy might be, instead of converting to a function with a value property, convert to something like `{ value, callTheUserObjectsOperation }`.

Reply:
Given the documentation, I'm OK with the current version of a function with an objectReference property. In my local version I renamed `converted` to `callTheUserObjectsOperation` to be even clearer.